src/ArcgisAccount/accountManagerUtils.js

```diff
@@ -322,10 +322,14 @@ const createAccountObject = async ({ dSession, portal, clientId }) => {
 
 const createAccountKey = ({ user, portal }) => {
   try {
+    const { customBaseUrl, portalHostname, urlKey } = portal || {};
+    const isAGOL = portalHostname === 'www.arcgis.com';
```

Comment:
@brandonlammey I wonder if `isPortal` is more reliable to use here?

https://developers.arcgis.com/rest/users-groups-and-items/portal-self.htm